sdks/python/README.md

```diff
@@ -118,7 +118,7 @@ set up your machine's Python development environment.
 #### Install ``pip``
 
 `pip` is Python's package manager.  If you already have `pip` installed
-(type `pip -V` to check), skip this step.
+(type `pip -V` to check), please make sure to have at least version 7.0.0.
```

Comment:
In that case `REQUIRED_PIP_VERSION = '0.7.0'` should change in the other file. I was confused that we are requiring 0.7.0 in the other file. The versions in both files should be the same.